src/vector/backends/numpy.py

```diff
@@ -251,7 +251,7 @@ def _toarrays(
     """
     istuple = True
     if not isinstance(result, tuple):
-        istuple = False
+        istuple = False  # type: ignore[unreachable]
```

Comment:
`type: ignore[unreachable]`

Reply:
Yeah, we just said it was a Tuple, now we are checking to see if it is. Not sure how "publicly" we want to let users know it accepts non Tuples; it technically should be `Union[Tuple[ScalarCollection, ...], ScalarCollection]``.